portal-kernel/test/unit/com/liferay/portal/kernel/servlet/ServletResponseUtilContentLengthTest.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.kernel.servlet;
+
+import com.liferay.portal.kernel.util.Props;
+import com.liferay.portal.kernel.util.PropsUtil;
+import com.liferay.portal.kernel.util.StreamUtil;
+
+import java.io.ByteArrayInputStream;
+
+import java.util.Arrays;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.mockito.Mockito;
+
+import org.springframework.mock.web.MockHttpServletResponse;
+
+/**
+ * @author Hai Yu
+ */
+public class ServletResponseUtilContentLengthTest {
+
+	@Test
+	public void testContentLengthLowerThanBufferSizeOfStreamUtil()
+		throws Exception {
+
+		PropsUtil.setProps(Mockito.mock(Props.class));
```

Comment:
@yuhai Since you don't need to mock any logic, you can just use a `ProxyFactory.newDummyInstance` here.